velox/docs/functions/spark/json.rst

```diff
@@ -11,3 +11,23 @@ JSON Functions
         SELECT json_object_keys(''); -- NULL
         SELECT json_object_keys(1); -- NULL
         SELECT json_object_keys('"hello"'); -- NULL
+
+.. spark:function:: from_json(jsonString) -> [json object]
+
+    Casting a JSON text to the function's output type returns the value 
+    represented by the JSON text if it matches the output type; otherwise, NULL 
+    is returned. 
+    The function supports ARRAY, MAP, and ROW as output types. For primitive 
```

Comment:
A bit ambiguous to me.
Suggestion:

Casts JSON string to ARRAY, MAP or ROW type. The output type is decided by the expression construction. If cast fails, returns NULL.